paddle/fluid/operators/conv_op.cc

```diff
@@ -412,18 +415,28 @@ framework::OpKernelType ConvOpGrad::GetExpectedKernelType(
                                  customized_type_value);
 }
 
+class NoOutputGradOpDescMaker
```

Comment:
Please refer our exists method on "scale_op", "batch_norm", we have tons of examples. Please do not create concepts.

Reply:
fixed